src/components/Select/Select.tsx

```diff
@@ -387,7 +388,7 @@ export const Select: FC<SelectProps> = React.forwardRef(
                             shape={selectShapeToTextInputShapeMap.get(shape)}
                             size={selectSizeToTextInputSizeMap.get(size)}
                             value={
-                                getSelectedOptionText() && !dropdownVisible
+                                !dropdownVisible
                                     ? getSelectedOptionText()
                                     : undefined
```

Comment:
does it matter at all that this changed the output from not including a value to including an empty value? I'm guessing no

Reply:
This check is only here to handle filterable multiple. We currently hide the input string when dropdown is visible here because it's a real bag of tricks to get the UX right. Planning to remove as soon as I figure out how to show the concatenated string, read from it and then update the selected items when the string is updated - probably using serialization.